modules/admin_manual/pages/installation/quick_guides/ubuntu_20_04.adoc

```diff
@@ -1,55 +1,56 @@
 = Install ownCloud on Ubuntu 20.04
 :toc: right
-:logrotate-url: https://linux.die.net/man/8/logrotate
-:webserver-user: www-data
-:webserver-group: www-data
-:install-directory: /var/www/owncloud
+:page-aliases: installation/quick_guides/ubuntu_18_04.adoc
```

Comment:
You need to remove the page-alias here (see drone error) because the page actually does exist. It was removed in master/10.9 because there is no longer support for it. The build should run fine afterwards.

Reply:
Okay